yarn.lock

```diff
@@ -9227,10 +9227,10 @@ __metadata:
   languageName: node
   linkType: hard
 
-"cookiejar@npm:^2.1.3":
-  version: 2.1.3
-  resolution: "cookiejar@npm:2.1.3"
-  checksum: 88259983ebc52ceb23cdacfa48762b6a518a57872eff1c7ed01d214fff5cf492e2660d7d5c04700a28f1787a76811df39e8639f8e17670b3cf94ecd86e161f07
+"cookiejar@npm:^2.1.4":
```

Comment:
Is this intentional, or was this to fix the failing `npm audit` test? (which might be resolved by #10551)

Reply:
It's related to https://github.com/bbc/simorgh/pull/10551.
I made the change to fix the failing `npm audit` test 
that reported https://github.com/advisories/GHSA-h452-7996-h45h .
I'll merge in `latest` and revert this change.
Thanks Karina.